application/src/main/java/org/opentripplanner/apis/transmodel/model/siri/et/SJEstimatedCallsType.java

```diff
@@ -0,0 +1,109 @@
+package org.opentripplanner.apis.transmodel.model.siri.et;
+
+import graphql.Scalars;
+import graphql.schema.DataFetchingEnvironment;
+import graphql.schema.GraphQLArgument;
+import graphql.schema.GraphQLFieldDefinition;
+import graphql.schema.GraphQLList;
+import graphql.schema.GraphQLNonNull;
+import graphql.schema.GraphQLObjectType;
+import graphql.schema.GraphQLTypeReference;
+import java.util.List;
+import org.opentripplanner.apis.transmodel.support.GqlUtil;
+import org.opentripplanner.model.TripTimeOnDate;
+
+public class SJEstimatedCallsType {
+
+  private static final String NAME = "SJEstimatedCalls";
+  public static final GraphQLTypeReference REF = new GraphQLTypeReference(NAME);
+
+  public static GraphQLObjectType create() {
+    return GraphQLObjectType.newObject()
+      .name(NAME)
+      .description(
+        "List of visits to quays as part of a vehicle journey, relative to the current quay and for a given date. Includes real-time updates"
```

Comment:
In other places in this type you talk about calls not visits. I think you'd want to be consistent.